src/test/resources/features/refapp-2.x/03-trackPatient/findPatient.feature

```diff
@@ -12,7 +12,10 @@ Feature: Patient Search
   @selenium
   @login
   Scenario: Searching an existing patient
-    And User enters John Smith
+    And User enters "<patientName>" patientName
     Then Search Page returns patients
     And User clicks on first patient
-    Then System loads patient dashboard
\ No newline at end of file
+    Then System loads patient dashboard
+     Examples:
+      | patientName  | 
+      | John Smith    | 
```

Comment:
Minor comment: I would adjust the last symbol(`I`) on the right hand side to be in vertical line with the one above it

Reply:
Resolved thnks